src/YesSql.Core/Store.cs

```diff
@@ -91,115 +91,105 @@ public async Task InitializeAsync()
 
             if (!string.IsNullOrEmpty(Configuration.Schema))
             {
-                await using (var connection = Configuration.ConnectionFactory.CreateConnection())
-                {
```

Comment:
Removing the code block means the connection will be disposed at the end of the method, and thus after the IdGenerator. That would lead to using more connections.

Reply:
added it back